src/backend/main/py.storage/caliopen_storage/core/base.py

```diff
@@ -150,3 +150,11 @@ def create(cls, user, **attrs):
         """Create a core object belong to an user."""
         obj = cls._model_class.create(user_id=user.user_id, **attrs)
         return cls(obj)
+
+    @classmethod
+    def object_belongs_to_user(cls, user_id, object_id):
```

Comment:
rename this method to belongs_to_user or something more pythonic, and you don't need to have a message_... object_... structure

so first remark on super disappear